src/components/home/ScheduleCard.tsx

```diff
@@ -0,0 +1,107 @@
+import { PropsWithChildren } from 'react'
+
+import {
+  useStyleConfig,
+  LinkBox,
+  LinkOverlay,
+  Flex,
+  Text,
+} from '@chakra-ui/react'
+
+interface Props {
+  title: string
+  startTime: Date
+  endTime?: Date
+  place?: string
+  placeLink?: string
+}
+
+const ScheduleCard = ({
+  title,
+  startTime,
+  endTime,
+  place,
+  placeLink,
+}: Props) => {
+  const startDay = startTime.toLocaleString('pt-BR', { day: '2-digit' })
+
+  const startMonth = startTime.toLocaleString('pt-BR', { month: 'long' })
+
+  const startHour = startTime.toLocaleTimeString('pt-BR', {
+    hour: '2-digit',
+    minute: '2-digit',
+  })
+
+  const endHour = endTime?.toLocaleTimeString('pt-BR', {
+    hour: '2-digit',
+    minute: '2-digit',
+  })
+
+  const isToday = (someDate: Date) => {
+    const today = new Date()
+    return (
+      someDate.getUTCDate() == today.getUTCDate() &&
+      someDate.getUTCMonth() == today.getUTCMonth() &&
+      someDate.getUTCFullYear() == today.getUTCFullYear()
+    )
+  }
+
+  interface StructureCardProps {
+    variant: string
+  }
+
+  const StructureCard = ({
+    variant,
```

Comment:
Tu pode definir um valor default no destructuring, pra caso a variant nn seja passada, assim:
```suggestion
    variant = 'base',
```

Ah, é importante que faz sentido também, na interface, colocar o variant como opcional (com `?`) quando a gente tem um valor default

Reply:
:done: